### DISCUSSION TITLE
Is there a way to view the foxglove-studio UI in a browser?

### DISCUSSION THREAD

**Discussion by MithunKinarullathil** [foxglove/community#80] [Q&A] [answered]
Hi, 

I was under the impression that the I can setup a server and if I go to an IP address I will be able to view the UI in a browser?. Is this possible? I tried adding the ip address using this command shown in the README but the UI does not show up at the address.

Thank you,
Mithun.

`yarn serve --host 192.168.xxx.yyy`

**Comment by MithunKinarullathil** [comment, marked as answer]
Solved:

Instead of 
`yarn serve` you can use `yarn web:serve` and go to the ip address in the first line.

**Reply by amacneil** [reply]
Yes, that's correct!
